bindinfo/handle.go

```diff
@@ -699,6 +712,7 @@ func GenerateBindSQL(ctx context.Context, stmtNode ast.StmtNode, planHint string
 	hint.BindHint(stmtNode, &hint.HintsSet{})
 	var sb strings.Builder
 	restoreCtx := format.NewRestoreCtx(format.DefaultRestoreFlags, &sb)
+	restoreCtx.DefaultDB = defaultDB
```

Comment:
Would this `DefaultDB` be used? Looks like the flag is not set?

Reply:
The `DefaultDB` is used in the parser.